README.md

```diff
@@ -1636,7 +1436,6 @@ See also [Are we web yet?](https://www.arewewebyet.org) and [Rust web framework
 
 A registry allows you to publish your Rust libraries as crate packages, to share them with others publicly and privately.
 
-* [Cloudsmith :heavy_dollar_sign:](https://cloudsmith.com/cargo-registry/) — A fully managed package management SaaS, with first-class support for public and private Cargo/Rust registries (plus many others). Has a generous free-tier and is also completely free for open-source.
```

Comment:
I'm not totally sure about this one. Evaluation of commercial things in this way is hard, but https://github.com/cloudsmith-io has only a handful of stars and no Rust code, so I'm kinda leaning towards the delete option.

Reply:
(I work there) Cloudsmith are supporting the ecosystem, though. We were the first service to support Cargo/Crates (outside of Crates.io) itself, as you can see here: https://cloudsmith.com/blog/worlds-first-private-cargo-registry-w-cloudsmith-rust/ - plus we've contributed to the ecosystem itself, and have customers (such as Embark, several -ifys, etc.) that also contribute. Although we're not personally built on Rust, it's something that might happen (e.g. static single dependency CLI).

Reply:
That's fair. I think this falls under the "Rust doesn't have much of this thing, so we've got slightly different standards for that area" criteria that's been noted for some other bits. Also, I can now document this in the comments about why we're overriding so that your entry gets kept in :)

I've re-added you.

Reply:
Very much appreciated, Tom, thank you! We'll continue to support the Rust (and other) ecosystems as best we can. :-)